db/modlogs.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Mongo = require(__dirname+'/db.js')
+	, db = Mongo.client.db('jschan').collection('modlog');
+
+module.exports = {
+
+	getFirst: () => {
```

Comment:
could be reduced to:

    getFirst: ()=>db.find({}).sort({_id:-1}).limit(1).toArray();